data/remnant missions.txt

```diff
@@ -495,3 +560,772 @@ mission "Remnant: Technology Available"
+			`	He shakes his head and continues, "We need to keep our medical facilities as free as possible. Could you take some of our injured to Aventine? They are stable and healthy enough for an unattended transport, and it would free up medics and space here."`
+			choice
+				`	"Of course."`
+				`	"I'd rather not."`
+					decline
+			`	He looks relieved. "Thanks. I was going to have to divert one of our sentries to take them." He pauses and consults his comlink. "The medical team will have the injured settled onboard shortly, along with everything they need to ensure they arrive safely. They will probably have some fresh crew to send back with you."`
+				accept
+	on stopover
+		event "Remnant Albatross"
+		dialog
+			`The injured crew make for an eerily quiet jaunt from Viminal to Aventine. They don't sing, and rarely sign. Some spend the trip mellowed out on powerful painkillers, while the few that leave their rooms seem to be mostly intent on doing prescribed stretches and movements to help damaged muscles heal. A couple are interested in visiting with someone from outside the Ember Waste, but they lack the energy to do so for long.`
+			`	On Aventine there are ambulances and medical staff waiting to transport them to longer-term medical facilities. As they depart, a fresh crew arrives to take their places.`
+	on complete
+		payment 100000
+		conversation
+			`The return trip to Viminal is much more interesting. The crew spend more time wandering your ship than they do in their bunks. Some spend most of their time down in engineering, while others hang out around the main guns or the cockpit. Only their remarkable aptitude at figuring out where to be in order to not interfere with normal operations keeps them from being a hindrance. As you open the hatch they disperse across the tarmac to the ships they have been assigned to, each one of them signing their thanks as they leave.`
+			`	A few minutes after they leave your comlink beeps, notifying you that a message from the prefect has arrived:`
+			`	"Captain <last>! Thank you for your help in defending Viminal and transporting those wounded. I have transferred a small token of our appreciation to you. That being said, perhaps you would appreciate this a bit more: the other prefects and I have agreed to grant you access to a wider range of ships. Our Albatross class warships, and Pelican class freighters will now be available to you. At the rate these raids are going, it is in everyone's best interests that you have access to a better warship."`
```

Comment:
There is a comma splice between "albatross" and "pelican". Also, I think the "class" extensions are unnecessary.

Reply:
I was aiming for a more formal style of presentation. As such, is the class distinction merely the sort of superfluous detail that someone might add to a ship introduction? Or actually wrong?

I consider that an important distinction, because while we may aim to have the writing of ES be perfect, we should not insist that characters within ES speak perfectly. And superfluous detail is something that people do.

That being said, I have replaced the whole phrase with

Our capital class ships will now be available to you, including the Albatross and the Pelican.